recipe_scrapers/littlesunnykitchen.py

```diff
@@ -0,0 +1,9 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
```

Comment:
Hi @traviswaelbro - thanks for suggesting this scraper!

There's a small cleanup opportunity here that I wasn't aware of, or had forgotten about -- we don't seem to need the `# mypy:` ignore filter for untyped functions any more.

```suggestion
from ._abstract import AbstractScraper
```

That's going to provide an opportunity for a _lot_ of cleanup elsewhere, but we don't need to worry about that during this pull request.